src/main/kotlin/racingcar/RacingCarApplication.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar
+
+import racingcar.domain.RacingCarGame
+import racingcar.strategy.RandomNumberGeneratorStrategy
+import racingcar.view.InputView
+import racingcar.view.ResultView
+
+class RacingCarApplication
+
+fun main() {
+    val userInput = InputView.getUserInput()
+    val racingCarGame = RacingCarGame(userInput, RandomNumberGeneratorStrategy())
```

Comment:
```suggestion
    val gameCondition = InputView.getUserInput()
    val racingCarGame = RacingCarGame(gameCondition, RandomNumberGeneratorStrategy())
```
사소할순 있지만, 가독성 측면에서  변수명도 신경써주셔도 좋아요 :)